js/utils/formUtils.ts

```diff
@@ -0,0 +1,17 @@
+import { FieldErrors } from 'react-hook-form';
```

Comment:
Denne filen kan legges under `MVP/utils(?)` mappen for kode vi har kontroll over.

- [ ] Flytt

- [ ]